rbd/mirror_test.go

```diff
@@ -17,6 +17,28 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestGetMirrorUUID(t *testing.T) {
+	conn := radosConnect(t)
```

Comment:
^^ @agarwal-mudit

Reply:
Hi @rajatsing  I am not sure what you were pointing out here to Mudit, but if you care to elaborate, perhaps I can help some more.